connectors/es/index.py

```diff
@@ -27,6 +27,16 @@ class TemporaryConnectorApiWrapper(ESClient):
     def __init__(self, elastic_config):
         super().__init__(elastic_config)
 
+    async def connector_get(self, connector_id, include_deleted):
+        include_deleted_param = (
+            f"?include_deleted={'true' if include_deleted else 'false'}"
```

Comment:
It took me way too long to realise python's bools are uppercased ....

Reply:
Is it possible to pass params as hash and do generic encoding in `perform_request`?

Reply:
Good callout! Yes it's [possible](https://github.com/elastic/elasticsearch-py/blob/14f22ae202d548caf9f6b620fb6fc73f1e224667/elasticsearch/_sync/client/_base.py#L312) and it will take care of lowercasing the bool ([reference](https://github.com/elastic/elasticsearch-py/blob/14f22ae202d548caf9f6b620fb6fc73f1e224667/elasticsearch/_sync/client/utils.py#L279))